### ISSUE TITLE
The Project is dead!

### ISSUE THREAD

**Issue by SaschaFitzner** [meetalva/alva#852]
I had contact with two former employees. 
It's a project by volunteers who left the company.

**Comment by offmarte** [comment]
well, that's sad. i'm new to programming but i'll try a shot at it later so i saved the source in my personal backup

**Comment by SaschaFitzner** [comment]
Yeah, it's sad, but the license is MIT an everybody can fork the project and start a new one :)

**Comment by daraul** [comment]
I literally just told myself alva was the prototyping tool for me :unamused:

Any ideas on a worthy successor?

**Comment by philippnagel** [comment]
@SaschaFitzner Would you be willing to fork the project? I am personally unable to commit that much time at the moment :/

**Comment by daraul** [comment]
I hadn't realized that alva is mostly typescript. It shouldn't be hard to keep this going through the community, but are the maintainers around to merge PRs?

**Comment by philippnagel** [comment]
@tilmx  ?
